### PR TITLE
Rework watch test

### PR DESCRIPTION
**Pull request by Meakk**
Needs https://github.com/f3d-app/f3d-docker-images/pull/26

### CODE REVIEW

File: application/testing/test_watch.sh [hunk 1]
```diff
@@ -21,10 +21,12 @@ pid=$!
 
 function cleanup()
 {
-  if [[ -z "${CTEST_F3D_COVERAGE-}" ]]; then
-    kill $pid
+  if [[ "$OSTYPE" == "linux-gnu"* ]]; then
+    wmctrl -c :ACTIVE:
```
Comment (mwestphal) [inline application/testing/test_watch.sh:25]: can we quit f3d instead of active window ?

File: application/testing/CMakeLists.txt [hunk 2]
```diff
@@ -1147,8 +1147,5 @@ if(WIN32 OR APPLE OR F3D_TESTING_ENABLE_GLX_TESTS) # Watch testing require GLX r
   endif()
   # Custom bash/pwsh test for testing watch option
   add_test (NAME f3d::TestWatch COMMAND ${_f3d_os_script_exec} ${CMAKE_CURRENT_SOURCE_DIR}/test_watch.${_f3d_os_script_ext} $<TARGET_FILE:f3d> ${F3D_SOURCE_DIR}/testing/data ${CMAKE_BINARY_DIR}/Testing/Temporary)
-  set_tests_properties(f3d::TestWatch PROPERTIES RUN_SERIAL TRUE)
-  if(F3D_COVERAGE) # When running test for coverage, adapt the script to use xdotool
-    set_tests_properties(f3d::TestWatch PROPERTIES ENVIRONMENT "CTEST_F3D_COVERAGE=1")
-  endif()
+  set_tests_properties(f3d::TestWatch PROPERTIES RUN_SERIAL TRUE TIMEOUT 60)
```
Comment (mwestphal) [inline application/testing/CMakeLists.txt:1150]: thats a high timeout, any reason why ?

Comment (Meakk) [reply application/testing/CMakeLists.txt:1150]: It was 1 hour before :P

Comment (mwestphal) [reply application/testing/CMakeLists.txt:1150]: ha you mean it was not defined and defaulted to 1h , ok!

Maybe make it lower ? I dont know how long this test takes on macOS

Comment (Meakk) [reply application/testing/CMakeLists.txt:1150]: I think it's fast. Around 7 seconds. I will lower it to 30s to be sure.

File: application/testing/test_watch.sh [hunk 3]
```diff
@@ -21,11 +21,7 @@ pid=$!
 
 function cleanup()
 {
-  if [[ -z "${CTEST_F3D_COVERAGE-}" ]]; then
-    kill $pid
-  else
-    xdotool key Escape
```
Comment (mwestphal) [inline application/testing/test_watch.sh:27]: lets remove xdotool and wmctrl from the images then.

Comment (Meakk) [reply application/testing/test_watch.sh:27]: I agree

**Review by mwestphal — APPROVED** [review]

**Pull request merged**